v63003/outref/gtm8889.txt

```diff
@@ -4,33 +4,23 @@
 
 spawn /usr/local/bin/tcsh -f
 > # Expect the shell prompt
-stty cols 132
-> YDB
+set shellprompt=SHELL
+> set prompt=$shellprompt
+SHELLstty cols 132
+SHELLYDB
 $ydb_dist/mumps -dir
-
 YDB>zhelp
-
-
-
 Additional information available: 
-
-About_YottaDB           Commands    Err_Processing          Functions
-Integrate_External      Internationalization    IO_Processing           
-ISV         Language_Extensions     M_Lang_Features         Opr_Dbg_Dir_Mode
-Program_Cycle           Triggers    Utility_Routines        
-
+About_GT.M  Commands    Err_Processing          Functions   Integrate_External
```

Comment:
Why has this output changed? For example, About_YottaDB changed to About_GT.M

Reply:
This change was a product of running the gtm8889 test with the latest version v999_r122. If this was meant to run with a different version, I can do that and change the outref file accordingly.